app/refactor/views/p4/tad/components/table_1/consignor_single.scala.xml

```diff
@@ -8,33 +8,33 @@
   <fo:table-cell display-align="center" padding-left="2mm">
     <fo:block>2 Consignor/Exporter</fo:block>
   </fo:table-cell>
-  <fo:table-cell display-align="center" padding-left="2mm">
+  <fo:table-cell number-columns-spanned = "3" display-align="center" padding-left="2mm">
     <fo:block>No &#160; <fo:inline font-weight = "bold">@vm.eori</fo:inline></fo:block>
   </fo:table-cell>
-  <fo:table-cell display-align="center" padding-left="2mm" font-weight="bold">
-    <fo:block>
-      @vm.streetAndNumber
-    </fo:block>
-  </fo:table-cell>
 </fo:table-row>
 <fo:table-row height="5.33mm">
-<fo:table-cell  display-align="center" padding-left="2mm" font-weight="bold">
+<fo:table-cell number-columns-spanned = "4" display-align="center" padding-left="2mm" font-weight="bold">
   <fo:block>
     @vm.name
   </fo:block>
 </fo:table-cell>
 </fo:table-row>
 <fo:table-row height="5.33mm">
-<fo:table-cell display-align="center" padding-left="2mm"  font-weight="bold">
+<fo:table-cell display-align="center" padding-left="2mm" font-weight="bold">
   <fo:block>
-    @vm.city
+    @vm.streetAndNumber
```

Comment:
are we concerned abut overflow?

Reply:
![image](https://github.com/hmrc/transit-movements-trader-manage-documents/assets/7061639/91a0d2ae-0b9f-47d4-86ec-f2da55ed2682)

Reply:
this is the old design, but @vajarova had this problem:
![image](https://github.com/hmrc/transit-movements-trader-manage-documents/assets/7061639/2fc010d0-17dd-4340-8970-1c7303cd444a)

Reply:
![image](https://github.com/hmrc/transit-movements-trader-manage-documents/assets/104501743/aa9e92fd-b33e-4ade-ae1c-9e2272e45af5)

Doesn't look too bad with the new format